c/src/ml-api-service-remote.c

```diff
@@ -563,9 +564,11 @@ ml_service_remote_release_internal (ml_service_s * mls)
 
   if (mlrs->edge_h) {
     nns_edge_release_handle (mlrs->edge_h);
+  }
```

Comment:
Waiting for release using sleep funciton has a risk of producing different results depending on the environment.
So I think it would be better to delete `g_usleep` like this 👍 

It's just a question, but I wonder if factors were considered that could cause side effects after removal sleep function.

Reply:
I think there is no need for sleep time because edge_h waits until the thread is destroyed using the join.
The use of sleep time seems an excessive defense code in the initial design.
Thanks.